content/en/users/tutorials/vm-datahub/_index.md

```diff
@@ -1,11 +1,110 @@
 ---
 title: "Virtual machine with DataHub access"
-linkTitle: "VM with DataHub acceess"
+linkTitle: "VM with DataHub access"
 type: docs
 weight: 30
-draft: true
 description: >
   Use data in EGI DataHub from a virtual machine
 ---
 
-TBD
+## Overview
+
+This tutorial describes the access to [EGI DataHub](../../datahub) spaces from a virtual
+machine. In the following paragraphs you will learn how to access data remotely stored in EGI DataHub via POSIX, by:
+
+- installing the oneclient component
+- configuring access to an EGI DataHub Oneprovider via oneclient
+
+## Prerequisites
+
+In order to access the EGI DataHub data you need, first of all, to have
+an [EGI Check-in](../../check-in) account, if you don't have an EGI
+Check-in account you can sign up [here](../../check-in/signup).
+
+## Oneclient installation
+
+The installation of oneclient package is currently supported for:
+
+- Ubuntu Bionic
+- Centos 7
+
+Work to support more recent versions of Operating Systems is ongoing, and alternatively
+a docker based installation is also provided.
+
+### Oneclient installation via packages
+
+Use the following command in order to install the oneclient package in a supported OS:
+
+```shell
+curl -sS http://get.onedata.org/oneclient.sh | bash
+```
+
+This will also install the needed dependencies.
+
+### Oneclient installation via docker
+
+In order to use the Dockerized version of oneclient (provided that you
+have docker installed), you can run the following  command:
+
+```shell
+docker run -it --privileged -v $PWD:/mnt/src --entrypoint bash onedata/oneclient:20.02.15
```

Comment:
```suggestion
$ docker run -it --privileged -v $PWD:/mnt/src --entrypoint bash onedata/oneclient:20.02.15
```

Reply:
While it may be handy for usage, I'm usually not fond of mixing short and long versions of options in documentation, and using only the long versions make the command more clear, something like below:
```shell
$ docker run --interactive --tty --privileged \
  --volume $PWD:/mnt/src \
  --entrypoint bash \
  onedata/oneclient:20.02.15
```